Tests/AmplitudeTests.m

```diff
@@ -1170,6 +1170,11 @@ - (void)testCustomizedLibraryWithNilLibraryAndVersion {
     XCTAssertEqualObjects(currentLibraryValue, targetLibraryValue);
 }
 
+<<<<<<< HEAD
+<<<<<<< Updated upstream
+=======
+=======
+>>>>>>> main
```

Comment:
Merge conflict?

Reply:
Arg, I accidently based this off of master instead of main
thanks